### PR TITLE
Feat/sync config files

### PR DESCRIPTION
**Pull request by jgaucher-cs**
Please ignore whitespaces when comparing changes: https://github.com/RS-PYTHON/rs-helm/pull/64/files?w=1

### CODE REVIEW

File: charts/rs-server-adgs/templates/configmap.yaml [hunk 1]
```diff
@@ -12,306 +12,345 @@
...
       {{ $k }}:
         auth:
           type: TokenAuth
-          auth_uri: '{{ $v.endpoint.url }}/oauth2/token'
+          auth_uri: {{ $v.endpoint.url }}/oauth2/token
           token_type: json
           token_key: access_token
-          request_method: POST        
+          request_method: POST
+          req_data:
```
Comment (jgaucher-cs) [inline charts/rs-server-adgs/templates/configmap.yaml:37]: @agrosu1978 @Padeanu is it normal that this req_data block is present in the rs-server config file but was missing from the helm charts ?

File: charts/rs-server-adgs/templates/configmap.yaml [hunk 2]
```diff
@@ -12,306 +12,345 @@
...
+          credentials:
+            username: test
+            password: test
+          headers:
+            Content-Type: application/x-www-form-urlencoded
         products:
           CAMS_GRF_AUX:
             productType: cams-ads-grf-aux
-
+          GENERIC_PRODUCT_TYPE:
```
Comment (jgaucher-cs) [inline charts/rs-server-adgs/templates/configmap.yaml]: @agrosu1978 @Padeanu same here for GENERIC_PRODUCT_TYPE

Comment (jgaucher-cs) [inline charts/rs-server-adgs/templates/configmap.yaml]: @agrosu1978 @Padeanu same here for GENERIC_PRODUCT_TYPE

File: charts/rs-server-adgs/templates/configmap.yaml [hunk 3]
```diff
@@ -12,306 +12,345 @@
...
           headers:
-              Authorization: "Bearer {token}"
+            Authorization: Bearer {token}
           credentials:
             token: token_value
         products:
           CAMS_GRF_AUX:
             productType: cams-ads-grf-aux
-
+          GENERIC_PRODUCT_TYPE:
```
Comment (jgaucher-cs) [inline charts/rs-server-adgs/templates/configmap.yaml]: @agrosu1978 @Padeanu same here

File: charts/rs-server-cadip/templates/configmap.yaml [hunk 4]
```diff
@@ -12,330 +12,330 @@
...
       {{ $k }}:
         auth:
           type: TokenAuth
-          auth_uri: '{{ $v.endpoint.url.root }}/oauth2/token'
+          auth_uri: {{ $v.endpoint.url.root }}/oauth2/token
           token_type: json
           token_key: access_token
-          request_method: POST          
+          request_method: POST
+          req_data:
```
Comment (jgaucher-cs) [inline charts/rs-server-cadip/templates/configmap.yaml:39]: @agrosu1978 @Padeanu same here

File: charts/rs-server-station-secrets/values.yaml [hunk 5]
```diff
@@ -181,4 +184,94 @@ app:
         password: test
         client_id: client_id
         client_secret: client_secret
-        authorization: Basic test
\ No newline at end of file
+        authorization: Basic test
+    cadip_session:
```
Comment (jgaucher-cs) [inline charts/rs-server-station-secrets/values.yaml]: @agrosu1978 @Padeanu why are the cadip "_session" stations defined in rs-demo/local-mode/config/rs-server.yaml but were missing from the helm charts ? Do we need them ? 

See the comment here: https://github.com/RS-PYTHON/rs-server/blob/develop/services/common/rs_server_common/authentication/authentication_to_external.py#L452
```python
# TODO: This part of the code has to be deleted after the cadip_ws_config(_token_module) files
# will be modified. The parameters from the cadip_session sections (ins, mti...) should replace those params
# from cadip sections (ins, mti...)
```
So what should we do with these ?

File: charts/rs-server-adgs/templates/configmap.yaml (hunk 2 above)
Comment (Padeanu) [reply charts/rs-server-adgs/templates/configmap.yaml]: GENERIC_PRODUCT_TYPE is no longer used, you can remove it (hopefully won't break anything)

File: charts/rs-server-station-secrets/values.yaml (hunk 5 above)
Comment (Padeanu) [reply charts/rs-server-station-secrets/values.yaml]: _sessions are defined in helm chart.

**Pull request merged**